.github/actions/e2e/slack/notify/action.yaml

```diff
@@ -18,8 +21,10 @@ runs:
     - id: get-run-name
       shell: bash
       run: |
-        if [[ ${{ github.event_name }} == "schedule" ]]; then
+        if [[ ${{ github.event_name }} == "schedule" && inputs.suite != "soak" ]]; then
           RUN_NAME="${{ inputs.suite }}-periodic"
+        elif [[ ${{ github.event_name }} == "schedule" && inputs.suite == "soak" ]]; then
```

Comment:
optional: You could make a simplifying assumption here since if the value doesn't equal soak, it will already be caught by the previous `if`. The way you have it is fine, but it's generally easier to start from more specific to less specific here when it comes to if conditionals, so something like this:

```bash
if [[ ${{ github.event_name }} == "schedule" && inputs.suite == "soak" ]]; then
  ....
elif [[ ${{ github.event_name }} == "schedule" ]]; then
  ....
else
  ....
fi
```